lib/puppet/provider/service/systemd.rb

```diff
@@ -59,7 +63,7 @@ def enabled?
       # flapping when simply trying to disable a masked service.
       return :mask if (@resource[:enable] == :mask) && (svc_info[:LoadState] == 'masked')
       return :true if svc_info[:UnitFileState] == 'enabled'
-      if Facter.value(:osfamily) == 'debian'
+      if Facter.value(:osfamily) == 'Debian'
```

Comment:
Would it be cleaner to just downcase the Facter.value instead of trying to make the RHS look right?